src/index.html

```diff
@@ -22,6 +22,361 @@
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#apple"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#samsung"
+                class="nav__link"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#smartphones"
+                class="nav__link"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#computers"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#gadgets"
+                class="nav__link"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#tablets"
+                class="nav__link"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#photo"
+                class="nav__link"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#video"
+                class="nav__link"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <main class="main">
+        <div
+          class="card-block"
+          data-qa="card"
+        >
+          <img
+            class="card-block__img"
+            src="images/imac.jpeg"
+            alt="iMac"
+          />
+          <h3 class="card-block__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+          <p class="card-block__code">Product code: 195434</p>
+          <div class="card-block__rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card-block__text">Reviews: 5</p>
+          </div>
+          <div class="card-block__price">
+            <p class="card-block__price_text">Price:</p>
+            <p class="card-block__price_number">$2,199</p>
+          </div>
+          <a
+            class="card-block__button"
+            data-qa="card-hover"
+          >
+            BUY
+          </a>
+        </div>
+        <div
```

Comment:
Add the blank line between the cards
```suggestion
        </div>
        
        <div
```